mava/components/executing/action_selection.py

```diff
@@ -96,7 +97,10 @@ def on_execution_select_actions(self, executor: SystemExecutor) -> None:
 
         observations = executor.store.observations
         # Normalize the observations before selecting actions.
-        if executor.store.global_config.normalize_observations:
+        if (
+            executor.has(ObservationNormalisation)
```

Comment:
Should we maybe create a BaseObservationNormalsiation component that we can extend from? Therefore if there is new normalisation implementations this if statement would still pass?